src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/MahojongTypes.cs

```diff
@@ -0,0 +1,240 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I seemed to recall we were trying to purge these Mahjong tests - are we still running them?